tests/unit/test_chainer.py

```diff
@@ -45,6 +45,15 @@
 GPU = 'ml.p2.xlarge'
 CPU = 'ml.c4.xlarge'
 
+ENDPOINT_DESC = {
```

Comment:
Can these just be empty dicts?

Reply:
Many of the tests still relies on mocking describe_endpoint returns 'EndpointName', so that's why I'm keeping the dictionary content and also to keep it consistent among predictor related rests.